tests/storager_bench_test.go

```diff
@@ -12,3 +16,23 @@ func BenchmarkStorage_Stat(b *testing.B) {
 		_, _ = s.Stat("abc")
 	}
 }
+
+func BenchmarkStorage_List(b *testing.B) {
+	var ob []*types.Object
+	for i := 0; i < 1024; i++ {
+		ob = append(ob, &types.Object{})
+	}
+
+	s := &Storage{
+		objects: ob,
+	}
+
+	it, err := s.list(nil, "", pairStorageList{})
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	for i := 0; i < b.N; i++ {
+		_, _ = it.Next()
```

Comment:
I'm confused about the target operation. Why is `Next` but not the complete list operation(retrieve all objects)?

Reply:
> I'm confused about the target operation. Why is `Next` but not the complete list operation(retrieve all objects)?

Because I want to measure the cost per `Next` operation.